core/boot/src/test/java/org/eclipse/dataspaceconnector/boot/system/ExtensionLoaderTest.java

```diff
@@ -68,6 +91,27 @@ void loadMonitor_whenNoMonitorExtension() {
         assertTrue(monitor instanceof ConsoleMonitor);
     }
 
+    @Test
+    void loadOpenTelemetry_whenNoOpenTelemetry() {
+        GlobalOpenTelemetry.set(new CustomOpenTelemetry());
+        var openTelemetry = ExtensionLoader.loadOpenTelemetry(new ArrayList<>());
+        assertEquals(openTelemetry, GlobalOpenTelemetry.get());
+    }
+
+    @Test
+    void loadOpenTelemetry_whenSingleOpenTelemetry() {
+        List<OpenTelemetry> openTelemetries = Arrays.asList(new CustomOpenTelemetry());
+        var openTelemetry = ExtensionLoader.loadOpenTelemetry(openTelemetries);
+        assertTrue(openTelemetry instanceof CustomOpenTelemetry);
```

Comment:
What about using assertj? The failure messages would be more descriptive
`assertThat(openTelemetry).isInstanceOf(CustomOpenTelemetry.class)`